packages/rulesets/src/spectral/test/latest-version-of-common-types-must-be-used.test.ts

```diff
@@ -0,0 +1,164 @@
+import { Spectral } from "@stoplight/spectral-core"
+import linterForRule from "./utils"
+import { LATEST_VERSION_BY_COMMON_TYPES_FILENAME } from "../functions/utils"
+
+let linter: Spectral
+
+beforeAll(async () => {
+  linter = await linterForRule("LatestVersionOfCommonTypesMustBeUsed")
+  return linter
+})
```

Comment:
This might make it so you don't have to filter out the other results:
```suggestion
let nonResolvingLinter: Spectral

beforeAll(async () => {
  nonResolvingLinter = await linterForRule("LatestVersionOfCommonTypesMustBeUsed", true)
  return nonResolvingLinter
})
```